.github/CONTRIBUTING.md

```
@@ -44,6 +29,42 @@ Prerequisites:
     indicates the version of python that needs to be installed for tox to run
     locally. In this case, the version of python that needs to be installed is
 
+## Linting and formatting
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)